client/src/pages/Availability.js

```diff
@@ -1,41 +1,82 @@
-import React, { useContext } from "react";
-import { withRouter } from "react-router-dom";
-import { Grid, Dialog, Hidden, Paper } from "@material-ui/core";
-import Menu from "../components/dashboard/Menu";
-import BackButton from "../components/dashboard/BackButton";
-import { UserContext } from "../context/Context";
+import React from "react";
+import { Box, Hidden, Paper, makeStyles } from "@material-ui/core";
+import SettingsMenu from "../components/dashboard/SettingsMenu";
 
-function Availability() {
-  const { dashboardDialogOpen } = useContext(UserContext);
+const useStyles = makeStyles(() => ({
+  centerPaper: {
+    display: "flex",
+    flexDirection: "column",
+    alignItems: "center",
+  },
+}));
 
+function Availability() {
+  const classes = useStyles();
   return (
-    <Grid container style={{ paddingTop: "90px", height: "100vh" }}>
-      <Grid item lg={3} md={3} sm={4} xs={12}>
-        <Menu />
-      </Grid>
-      <Hidden xsDown>
-        <Grid item lg={9} md={9} sm={8} style={{ padding: "15px" }}>
-          <Paper square elevation={3} style={{ width: "100%", height: "100%" }}>
-            Availability
+    <Box
+      display="flex"
+      flexWrap="nowrap"
+      style={{ paddingTop: "90px", minHeight: "100vh" }}
+      bgcolor="grey"
+    >
+      <Hidden smDown>
+        <Box>
+          <SettingsMenu />
+        </Box>
+      </Hidden>
+      <Hidden smDown>
```

Comment:
Instead of using these Hidden components and repeating the component structure, look into using [breakpoints](https://material-ui.com/customization/breakpoints/) inside your makeStyles function. That should simplify your code quite a bit